src/TestUtils/src/DeviceTests/AssertionExtensions.iOS.cs

```diff
@@ -273,7 +273,7 @@ public static UIImage AssertContainsColor(this UIImage bitmap, UIColor expectedC
 			return bitmap;
 		}
 
-		public static Task AssertEqual(this UIImage bitmap, UIImage other)
+		public static Task AssertEqualAsync(this UIImage bitmap, UIImage other)
```

Comment:
Does this method need the suffix? It doesn't appear to be awaitable?

Reply:
I think it's so that the APIs are symmetrical between platforms. This API is awaitable on `Windows` so anywhere this is used in an xplat way needs to have the same signature.